backend/timed/tracking/serializers.py

```diff
@@ -68,6 +69,21 @@ def validate_running_activity():
 
         return data
 
+    def update(
+        self, instance: models.Activity, validated_data: dict
+    ) -> models.Activity:
+        """Update an activity.
+
+        Ensure that transferring an activity ends it.
+        """
+        transferred = validated_data.pop("transferred", instance.transferred)
```

Comment:
Honest question: What even is a transferred activity? And why would we `pop()` the value here? This would imply that we'll never set that attribute again (as it's not in `validated_data` anymore after this line)

Reply:
AFAICT the frontend transfers an activity but leaves it running resulting in https://github.com/adfinis/timed/issues/233

Reply:
an activity gets set to transferred when you e.g. generate a timesheet, after that you can no longer edit it

Reply:
```py
    permission_classes = (
        (
            # users may not change transferred activities
            IsAuthenticated & IsInternal & IsNotTransferred
            | IsAuthenticated & IsReadOnly
            # only external employees with resource role may create not transferred activities
            | IsAuthenticated & IsExternal & IsResource & IsNotTransferred
        ),
    )
```

Reply:
> an activity gets set to transferred when you e.g. generate a timesheet, after that you can no longer edit it

That makes sense, thanks! Found the permission code, but not what is causing it.

In that case, you probably still shouldn't `pop()` the value tho (use `.get()` instead)